### DISCUSSION TITLE
AdMob - nonPersonalizedAds - Question

### DISCUSSION THREAD

**Discussion by badranmo** [firebase/flutterfire#3528] [Q&A]
Hello, 
For the firebase_admob package -  how does "nonPersonalizedAds: true" work in MobileAdTargetingInfo class? Does it set all ads to all users everywhere to non personalized or only those in the UK and EU? 
If the Admob  account settings are set to personalized ads and MobileAdTargetingInfo class is set to nonPersonalized ads , will ALL users get non personalized ads or only those in the EU and UK? I couldn't see a clear explanation in the docs.
Thanks

**Comment by abdullah432** [comment]
Any solution?
